src/scripts/i18n/i18n_extraction.py

```
@@ -0,0 +1,364 @@
+import concurrent.futures
+import argparse
+from dotenv import load_dotenv
+
+load_dotenv(dotenv_path=os.path.join(os.path.dirname(__file__), '.env'))
+
+# Get Gemini API key from environment variable
+GEMINI_API_KEY = os.environ.get('GEMINI_API_KEY')
+if not GEMINI_API_KEY:
+    raise RuntimeError('GEMINI_API_KEY environment variable not set')
+GEMINI_API_URL = (
+    f'https://generativelanguage.googleapis.com/v1beta/models/gemini-2.5-pro-preview-06-05:generateContent?key={GEMINI_API_KEY}'
```

Pylint (3.3.7)

[convention] Line too long (128/100)

(C0301)